packages/kit-bg/src/services/ServiceFirmwareUpdate/ServiceFirmwareUpdate.ts

```
@@ -1536,6 +1602,78 @@
+          installingTarget: {} as any,
+          // totalPhase: workflowParams.releaseResult.totalPhase,
+          // currentPhase: firmwareType,
+          // updateInfo,
+        },
+      });
+
+      const convertVersion = (version?: string) => {
+        if (version && semver.valid(version)) {
+          return version.split('.').map((v) => parseInt(v, 10));
+        }
+        return undefined;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)